Source/Atomic/Resource/BackgroundLoader.cpp

```diff
@@ -264,20 +264,12 @@ void BackgroundLoader::FinishBackgroundLoading(BackgroundLoadItem& item)
     // If BeginLoad() phase was successful, call EndLoad() and get the final success/failure result
     if (success)
     {
-#ifdef ATOMIC_PROFILING
+#if ATOMIC_PROFILING
```

Comment:
On these profiling changes, I don't think we need to pepper ATOMIC modification blocks everywhere, we've diverged... next update, I guess see what run into and maybe adjust, though think modification blocks would just add noise in these cases